example-tool.yaml

```diff
@@ -0,0 +1,16 @@
+# Tool request example file.
+
+# Currently we only support gcloud cli, both `do` and `cleanup` commands need to
+# be without the cli name.
+
+# See below example:
+
+do:
```

Comment:
Don't we need to specify the tool?

Reply:
default is gcloud.

Reply:
I think it's better to have the explicit value here. And comment that the default value is gcloud

Reply:
Sure.